src/containers/Home/index.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import { useTypedSelector } from '../../store';
 
 const Home: React.FC = () => {
-  return <h2>Home</h2>;
+  const docType = useTypedSelector(state => state.createDoc.docType);
```

Comment:
Why do we need this on the Home component ? is just a reference of how to get something out of the store  ?